Db recap2020 (#140)
dot-layer/blog: content/blog/2021-02-25-recap-2020/index.fr.md

---
title: .Layer en 2020
author: Samuel Perreault et David Beauchemin
date: '2021-02-25'
slug: recap-2020
type: post
categories: ["Nouvelles .Layer"]
tags: []
description: ""
featured: "recap-2020-cover.png"
featuredpath: "img/headers/"
---

Sans surprise pour personne, 2020 a été une année différente pour .Layer. Malgré cela, 2020 fut une année remplie pour .Layer. Comme on dit, "vaux mieux tard que jamais", donc voici donc un petit *recap* pour la communauté.

# Administration de l'organisme sans but lucratif (OSBL) 

> La crise planétaire de 2020 ne nous a pas empêchés de tenir notre assemblée générale annuelle (AGA) dans un format en ligne. Durant l'AGA, plusieurs points importants ont été discutés, tels que la charte, le bilan financier, les projets et surtout la notion de membre actif.

Reviewer: ```suggestion
La crise planétaire de 2020 ne nous a pas empêché de tenir notre assemblée générale annuelle (AGA) dans un format en ligne. Durant l'AGA, plusieurs points importants ont été discutés, tels que la charte, le bilan financier, les projets et surtout la notion de membre actif.
```

- Droit de vote en assemblée pour les élections des administrateurs.
- Être prioritaire dans les événements .Layer (dans le sens de prioritaire pour la réservation de sa place et non pas d'être `VIP`).
- _Bragger_ que tu supportes les activités d'une OSBL en science des données (probablement le plus cool de la liste).

Une dizaine de personnes ont effectué les démarches pour devenir membres actifs (voir la liste [ici](https://github.com/dot-layer/charte-osbl/blob/master/LISTEMEMBRES.md)). Il est possible de s'y joindre en suivant cette [procédure](https://github.com/dot-layer/charte-osbl/blob/master/DEVENIRMEMBRE.md) qui se résume essentiellement à écrire à `david.beauchemin.5@ulaval.ca` et à faire un virement interac de 20$. 

# Comité blogue
Le comité blogue a connu une bonne première année d'existence.  Ils ont supervisé la publication de 9 articles en 2020. Plusieurs améliorations esthétiques ont aussi été mises en place par le comité et surtout par [Annie Deshaies](https://github.com/AnnieDeshaies) (merci Annie, for real merci!).

> Ont rappel que le blogue est ouvert à toutes sortes d’articles : français, anglais, vulgarisation de concepts (apprentissage machine, programmation, science des données, statistiques, etc.), expérience personnelle, présentation d’un projet, name it! Ne soyez pas timide. En parlant de ces fameux articles, en voici la liste.

Reviewer: ```suggestion
On rappel que le blogue est ouvert à toutes sortes d’articles : français, anglais, vulgarisation de concepts (apprentissage machine, programmation, science des données, statistiques, etc.), expérience personnelle, présentation d’un projet, name it! Ne soyez pas timide. En parlant de ces fameux articles, en voici la liste.
```

- [Training a Recurrent Neural Network (RNN) using PyTorch](https://www.dotlayer.org/en/blog/2020-08-19-train-a-sequence-model-with-poutyne/machine-learning/)
- [Reproductibilité en apprentissage automatique](https://www.dotlayer.org/blog/2020-10-30-reproducibility-in-ml-a-talk/machine-learning/)
- [OpenLayer, 1 an plus tard](https://www.dotlayer.org/blog/2020-07-29-openlayer-one-year/podcast/)
- [Kobenhavn](https://www.dotlayer.org/blog/2020-06-09/copenhagen/)
- [Comment proposer un article](https://www.dotlayer.org/blog/2020-03-19-howto/howto-fr/)
- [Boosting Trees avec Julia](https://www.dotlayer.org/blog/2020-01-04-julia-boosting-trees/julia-boosting-trees/)
- [What's wrong with scikit-learn](https://www.dotlayer.org/en/blog/2020-03-04-what-is-wrong-with-sklearn/neat-machine-learning-pipelines/)
- [Recap 2019](https://www.dotlayer.org/blog/2019-12-19-recap-2019/recap-2019/)
- [Argparse: un outil méconnu](https://www.dotlayer.org/blog/2019-05-21-argparse-package/optparse-package/)

## Projets futurs

> Pour 2021, le comité vise un plus grand nombre de publications, soit une douzaine. De plus, cela fait [un moment](https://github.com/dot-layer/blog/issues/14) que nous parlons d'ajouter une bibliothèque de critique (review) de livre, en espérant que cette année sera la bonne! Finalement, nous avons décidé d'ajouter une section [nos membres](https://github.com/dot-layer/blog/issues/112) afin de remercier les membres actifs qui supportent l'initiative. Bien sûr, si vous avez des idées d'améliorations vos suggestions sont très appréciées via l'ouverture d'une [issue](https://github.com/dot-layer/blog/issues).

Reviewer 2: ```suggestion
Pour 2021, le comité vise un plus grand nombre de publications, soit une douzaine. De plus, cela fait [un moment](https://github.com/dot-layer/blog/issues/14) que nous parlons d'ajouter une bibliothèque de critiques (reviews) de livre, en espérant que cette année sera la bonne! Finalement, nous avons décidé d'ajouter une section [nos membres](https://github.com/dot-layer/blog/issues/112) afin de remercier les membres actifs qui supportent l'initiative. Bien sûr, si vous avez des idées d'améliorations vos suggestions sont très appréciées. Pour ce faire, la meilleure façon est de créer une [issue](https://github.com/dot-layer/blog/issues) sur le dépôt du blogue.
```

# OpenLayer

> En 2019, [David Beauchemin](https://github.com/davebulaval) a introduit [OpenLayer](https://anchor.fm/open-layer), un podcast sur l'IA et l'apprentissage machine. Puisque la formule est en présentiel, le nombre d'épisodes publié a été plus bas que ce qui était prévu (27 de moins que prévu), soit 13 pour un total de 32 épisodes total. Toutefois, la formule a connu un grand succès avec pas loin de 5 000 vues sur YouTube et 2 000 sur les autres réseaux de diffusion (c.-à-d. Spotify et iTunes). Cette pause forcée a permis toutefois d'améliorer l'esthétique du podcast avec l'ajout de _thumbnails_, l'amélioration des messages de diffusion et le peaufinage de l'image. 

Reviewer 2: ```suggestion
En 2019, [David Beauchemin](https://github.com/davebulaval) a introduit [OpenLayer](https://anchor.fm/open-layer), un podcast sur l'IA et l'apprentissage machine. Puisque la formule est en présentiel, le nombre d'épisodes publiés a été plus bas que prévu, soit 13 (au lieu de 40). Ceci donne tout de même un total de 32 épisodes pour le podcast! La formule a néanmoins connu un grand succès avec près de 5 000 vues sur YouTube et 2 000 sur les autres réseaux de diffusion (c.-à-d. Spotify et iTunes). Cette pause forcée a toutefois permis d'améliorer l'esthétique du podcast grâce à l'ajout de _thumbnails_, l'amélioration des messages de diffusion et le peaufinage de l'image. 
```

> En 2019, [David Beauchemi](https://github.com/davebulaval) a introduit [OpenLayer](https://anchor.fm/open-layer), un podcast sur l'IA et l'apprentissage machine. Puisque la formule est en présentiel, le nombre d'épisodes publié a été plus bas que ce qui était prévu (27 de moins que prévu), soit 13 pour un total de 32 épisodes total. Toutefois, la formule a connu un grand succès avec pas loin de 5 000 vues sur YouTube et 2 000 sur les autres réseaux de diffusion (c.-à-d. Spotify et iTunes). Cette pause forcée a permis toutefois d'améliorer l'esthétique du podcast avec l'ajout de _thumbnails_, l'amélioration des messages de diffusion et le peaufinage de l'image. 

Reviewer: ```suggestion
En 2019, [David Beauchemin](https://github.com/davebulaval) a introduit [OpenLayer](https://anchor.fm/open-layer), un podcast sur l'IA et l'apprentissage machine. Puisque la formule est en présentiel, le nombre d'épisodes publié a été plus bas que ce qui était prévu (27 de moins que prévu), soit 13 pour un total de 32 épisodes total. Toutefois, la formule a connu un grand succès avec pas loin de 5 000 vues sur YouTube et 2 000 sur les autres réseaux de diffusion (c.-à-d. Spotify et iTunes). Cette pause forcée a permis toutefois d'améliorer l'esthétique du podcast avec l'ajout de _thumbnails_, l'amélioration des messages de diffusion et le peaufinage de l'image. 
```

Author: RIP

## Vision future

> Pour 2021, puisque la situation ne s'améliore pas et que la formule habituelle de le faire en personne n'est pas encore possible, une série _spéciale_ en ligne, en anglais, commence à être mise sur pied. L'idée est de "profiter" de l'impossibilité d'être en personne pour échanger avec des gens de l'extérieur du Québec. Avec cette approche, le podcast deviendrait bilingue.

Reviewer 2: Y'a de quoi à revoir avec la fin du chunk.
```suggestion
Pour 2021, puisque la situation ne s'améliore pas et que la formule *en personne* habituelle n'est pas encore possible, une série _spéciale_ en ligne, en anglais, commence à être mise sur pied. L'idée est de "profiter" de l'impossibilité d'être en personne pour échanger avec des gens de l'extérieur du Québec. Avec cette approche, le podcast deviendrait bilingue.
```

# Meetup Machine Learning Québec
![Meetup chez Can-Explore](fig/merge_img_can_explore.jpg)
Crédit photo : Can-Explore

> En 2019 nous avions dit que "2020 s'annonce tout aussi occupée avec un meetup déjà *in the making*. On vise aussi à incorporer plus de formules 5@7 et potentiellement un hackaton."

Reviewer 2: ```suggestion
En 2019, nous écrivions : "2020 s'annonce tout aussi occupée [que 2019] avec un meetup déjà *in the making*. On vise aussi à incorporer plus de formules 5@7 et potentiellement un hackaton."
```

_I guess not!_

> Finalement, nous avons _**seulement**_ eu le droit à un évènement meetup le 12 février chez Can-Explore. L'évènement fut un grand succès avec une cinquantaine de personnes. La gang de Meetup ML n'a pas décidé de faire d'évènement en ligne puisque la formule est très axée sur les échanges pendant et après les présentations. L'évènement nous semblait dénaturé de son essence et nous avons préféré concentrer nos efforts sur autre chose comme avancer nos 10 autres projets. Nous restons bien sûr à l'affut de tout changement et espérons en organiser un vers la fin de 2021.

Reviewer 2: Ça commence à la troisième personne (sing.), ça finit à la deuxième (plur.).
```suggestion
Finalement, nous avons _**seulement**_ eu droit à un évènement meetup, le 12 février chez Can-Explore. L'évènement fut un grand succès avec une cinquantaine de personnes (le bon vieux temps quoi). La gang de Meetup ML a décidé de ne pas organiser d'évènement en ligne puisque la formule originale est très axée sur les échanges pendant et après les présentations; l'évènement en ligne nous semblait dénaturé. Nous avons préféré concentrer nos efforts sur autre chose : nos 10 autres projets. Nous restons bien sûr à l'affut de tout changement et espérons organiser meetup vers la fin de 2021.
```

> Finalement, nous avons _**seulement**_ eu le droit à un évènement meetup le 12 février chez Can-Explore. L'évènement fut un grand succès avec une cinquantaine de personnes. La gang de Meetup ML n'a pas décidé de faire d'évènement en ligne puisque la formule est très accès sur les échanges pendant et après les présentations. L'évènement nous semblait dénaturaliser de son essence et nous avons préféré concentrer nos efforts sur autre chose comme avancer nos 10 autres projets. Nous restons bien sûr à l'affut de tout changement et espérons en organiser un vers la fin de 2021.

Reviewer: ```suggestion
Finalement, nous avons _**seulement**_ eu le droit à un évènement meetup le 12 février chez Can-Explore. L'évènement fut un grand succès avec une cinquantaine de personnes. La gang de Meetup ML n'a pas décidé de faire d'évènement en ligne puisque la formule est très axée sur les échanges pendant et après les présentations. L'évènement nous semblait dénaturé de son essence et nous avons préféré concentrer nos efforts sur autre chose comme avancer nos 10 autres projets. Nous restons bien sûr à l'affut de tout changement et espérons en organiser un vers la fin de 2021.
```

# Stage sur le projet Universal Data Tool.

> Pour 2020, nous avons eu un stagiaire en programmation durant 5 semaines. Il a travaillé sur le développement de fonctionnalité pour l'outil d'annotation de données [Universal Data Tool](https://universaldatatool.com/).Son travail a permis l'intégration de [plusieurs fonctionnalités](https://github.com/UniversalDataTool/universal-data-tool/issues?q=CedricJean) tel que la [gestion Cognito S3](https://github.com/UniversalDataTool/universal-data-tool/pull/103).

Reviewer 2: Ça c'est pas clair pour moi... C'est un stagiaire .Layer?
```suggestion
Aussi en 2020, nous avons eu un stagiaire en programmation durant 5 semaines. Il a travaillé sur le l'outil d'annotation de données [Universal Data Tool](https://universaldatatool.com/). Son travail a permis l'intégration de [plusieurs fonctionnalités](https://github.com/UniversalDataTool/universal-data-tool/issues?q=CedricJean) telle que la [gestion Cognito S3](https://github.com/UniversalDataTool/universal-data-tool/pull/103).
```

> Mentionnons [Marc-Alexandre Paquet](https://github.com/Ownmarc) qui a eu (1) l'idée et (2) qui a fortement contribué à la supervision du stagiaire durant son stage.

Reviewer 2: ```suggestion
Mentionnons [Marc-Alexandre Paquet](https://github.com/Ownmarc) qui a eu (1) l'idée et (2) qui a fortement contribué à la supervision du stagiaire.
```

---
title: .Layer en 2020
author: Samuel Perreault et David Beauchemin
date: '2021-02-25'
slug: recap-2020
type: post
categories: ["Nouvelles .Layer"]
tags: []
description: ""
featured: "recap-2020-cover.png"
featuredpath: "img/headers/"
---

> Ce n'est une surprise pour personne, 2020 a été une année différente pour .Layer. Elle fut néanmoins une année assez remplie. Voici donc un petit *recap* pour la communauté. Comme on dit, "vaux mieux tard que jamais"!

Author: ```suggestion
Ce n'est une surprise pour personne, 2020 a été une année différente pour .Layer. Elle fut néanmoins une année assez remplie. Voici donc un petit *recap* pour la communauté. Comme on dit, « vaux mieux tard que jamais »!
```

# Administration de l'organisme sans but lucratif (OSBL) 
La crise planétaire de 2020 ne nous a pas empêché de tenir notre assemblée générale annuelle (AGA) dans un format en ligne. Durant l'AGA, plusieurs points importants ont été discutés, tels que la charte, le bilan financier, les projets en cours et futurs, et surtout la notion de membre actif.

Ce dernier point, celui des membres actifs, était sans aucun doute le dernier morceau à mettre en place pour finaliser la structure de l'OSBL. Deux éléments motivent cette initiative : (1) couvrir les frais annuels liés à l'OSBL et (2) fournir un petit coussin pour les imprévus. En devenant membre membre actif, un individu obtiendra les avantages suivants :

- le droit de vote en assemblée lors de l'élection des administrateurs;
- la priorité de réservation pour les événements .Layer;
- _Bragger_ que tu supportes les activités d'une OSBL en science des données (de loin le plus cool de la liste).

> Une dizaine de personnes ont effectué les démarches pour devenir membres actifs (voir la liste [ici](https://github.com/dot-layer/charte-osbl/blob/master/LISTEMEMBRES.md)). Il est possible de s'y joindre en suivant cette [procédure](https://github.com/dot-layer/charte-osbl/blob/master/DEVENIRMEMBRE.md) qui se résume essentiellement à écrire à `jtbaillargeon@hotmail.com` ton nom complet avec et à faire un virement interac de 20$ au trésorier (JTBai). 

Author: ```suggestion
Une dizaine de personnes ont effectué les démarches pour devenir membres actifs (voir la liste [ici](https://github.com/dot-layer/charte-osbl/blob/master/LISTEMEMBRES.md)). Il est possible de s'y joindre en suivant cette [procédure](https://github.com/dot-layer/charte-osbl/blob/master/DEVENIRMEMBRE.md) qui se résume essentiellement à écrire à `jtbaillargeon@hotmail.com` ton nom complet et à faire un virement interac de 20$ au trésorier (JTBai). 
```

# Comité blogue
Le comité blogue a connu une bonne première année d'existence.  Ils ont supervisé la publication de 9 articles en 2020. Plusieurs améliorations esthétiques ont aussi été mises en place par le comité et surtout par [Annie Deshaies](https://github.com/AnnieDeshaies) (merci Annie, for real merci!).

> On rappelle que le blogue est lieu de publication pour toutes sortes d’articles : français, anglais, vulgarisation de concepts (apprentissage machine, programmation, science des données, statistiques, etc.), expérience personnelle, présentation d’un projet, *name it*! Ne soyez pas timide. En parlant de ces fameux articles, en voici la liste :

Reviewer 3: ```suggestion
On rappelle que le blogue est lieu de publication (en français ou en anglais) pour des articles portant sur des sujets variés  : vulgarisation de concepts (apprentissage machine, programmation, science des données, statistiques, etc.), expérience personnelle, présentation d’un projet, *name it*! Ne soyez pas timide. En parlant de ces fameux articles, en voici la liste :
```

- [Training a Recurrent Neural Network (RNN) using PyTorch](https://www.dotlayer.org/en/blog/2020-08-19-train-a-sequence-model-with-poutyne/machine-learning/)
- [Reproductibilité en apprentissage automatique](https://www.dotlayer.org/blog/2020-10-30-reproducibility-in-ml-a-talk/machine-learning/)
- [OpenLayer, 1 an plus tard](https://www.dotlayer.org/blog/2020-07-29-openlayer-one-year/podcast/)
- [Kobenhavn](https://www.dotlayer.org/blog/2020-06-09/copenhagen/)
- [Comment proposer un article](https://www.dotlayer.org/blog/2020-03-19-howto/howto-fr/)
- [Boosting Trees avec Julia](https://www.dotlayer.org/blog/2020-01-04-julia-boosting-trees/julia-boosting-trees/)
- [What's wrong with scikit-learn](https://www.dotlayer.org/en/blog/2020-03-04-what-is-wrong-with-sklearn/neat-machine-learning-pipelines/)
- [Recap 2019](https://www.dotlayer.org/blog/2019-12-19-recap-2019/recap-2019/)
- [Argparse: un outil méconnu](https://www.dotlayer.org/blog/2019-05-21-argparse-package/optparse-package/)

## Projets futurs

> Pour 2021, le comité vise un plus grand nombre de publications, soit une douzaine. De plus, cela fait [un moment](https://github.com/dot-layer/blog/issues/14) que nous parlons d'ajouter une bibliothèque de critiques (reviews) de livre, en espérant que cette année sera la bonne! Finalement, nous avons décidé d'ajouter une section [nos membres](https://github.com/dot-layer/blog/issues/112) afin de remercier les membres actifs qui supportent l'initiative. Bien sûr, si vous avez des idées d'améliorations vos suggestions sont très appréciées. Pour ce faire, la meilleure façon est de créer une [issue](https://github.com/dot-layer/blog/issues) sur le dépôt du blogue.

Reviewer 3: ```suggestion
Pour 2021, le comité vise un plus grand nombre de publications, soit une douzaine. De plus, cela fait [un moment](https://github.com/dot-layer/blog/issues/14) que nous parlons d'ajouter une bibliothèque de critiques (reviews) de livre, en espérant que cette année sera la bonne! Finalement, nous planifions aussi d'ajouter une section [nos membres](https://github.com/dot-layer/blog/issues/112) afin de remercier les membres actifs qui supportent l'initiative. Bien sûr, si vos idées/suggestions sont toujours très appréciées. Pour ce faire, la meilleure façon est de créer une [issue](https://github.com/dot-layer/blog/issues) sur le dépôt du blogue.
```

## Vision future
Pour 2021, puisque la situation ne s'améliore pas et que la formule *en personne* habituelle n'est pas encore possible, une série _spéciale_ en ligne, en anglais, commence à être mise sur pied. L'idée est de "profiter" de l'impossibilité d'être en personne pour échanger avec des gens de l'extérieur du Québec.
# Meetup Machine Learning Québec
![Meetup chez Can-Explore](fig/merge_img_can_explore.jpg)
Crédit photo : Can-Explore

En 2019, nous écrivions : "2020 s'annonce tout aussi occupée [que 2019] avec un meetup déjà *in the making*. On vise aussi à incorporer plus de formules 5@7 et potentiellement un hackaton."

_I guess not!_

> Finalement, nous avons _**seulement**_ eu droit à un évènement meetup, le 12 février chez Can-Explore. L'évènement fut un grand succès avec une cinquantaine de personnes (le bon vieux temps quoi). La gang de Meetup ML a décidé de ne pas organiser d'évènement en ligne puisque la formule originale est très axée sur les échanges pendant et après les présentations; l'évènement en ligne nous semblait dénaturé. Nous avons préféré concentrer nos efforts sur autre chose : nos 10 autres projets. Nous restons bien sûr à l'affut de tout changement et espérons organiser meetup vers la fin de 2021.

Reviewer 3: ```suggestion
Finalement, nous avons _**seulement**_ eu droit à un évènement meetup, le 12 février chez Can-Explore. L'évènement fut un grand succès avec une cinquantaine de personnes (le bon vieux temps quoi). La gang de Meetup ML a décidé de ne pas organiser d'évènement en ligne puisque la formule originale est très axée sur les échanges pendant et après les présentations; l'évènement en ligne nous semblait dénaturé. Nous avons préféré concentrer nos efforts sur autre chose : nos 10 autres projets. Nous restons bien sûr à l'affut de tout changement et espérons organiser un meetup vers la fin de 2021.
```

> # Stage sur le projet Universal Data Tool.

Reviewer 3: ```suggestion
# Stage sur le projet Universal Data Tool
```

Officiellement, .Layer a comme mission de promouvoir la collaboration et le partage de connaissances dans le domaine de la science des données. Ceci veut dire, entre autres choses, d'appuyer (1) des stagiaires dans leur développement et (2) des projets *open-source*. On souligne donc l'initiative de [Marc-Alexandre Paquet](https://github.com/Ownmarc) (qui a pondu cette idée de génie) et David Beauchemin (on s'en sort pas, encore lui) qui ont utilisé le statut d'OSBL de .Layer pour offrir un stage en programmation de cinq semaines à Cedric Jean. Cedric a travaillé sur l'outil d'annotation de données [Universal Data Tool](https://universaldatatool.com/). Son travail a permis l'intégration de [plusieurs fonctionnalités](https://github.com/UniversalDataTool/universal-data-tool/issues?q=CedricJean) telle que la [gestion Cognito S3](https://github.com/UniversalDataTool/universal-data-tool/pull/103).

> Message aux membres .Layer, si vous êtes intéressés à encadrer un étudiant (pour un projet lié aux valeur de l'organisation), mais qu'il vous manque un cadre formel pour officialiser le tout, n'hésitez pas à en parler à un membre du CA.

Reviewer 3: ```suggestion
Message aux membres .Layer, si vous êtes intéressés à encadrer un étudiant (pour un projet lié aux valeurs de l'organisation), mais qu'il vous manque un cadre formel pour officialiser le tout, n'hésitez pas à en parler à un membre du CA.
```
